backbone.layoutmanager.js

```diff
@@ -385,11 +385,19 @@ var LayoutManager = Backbone.View.extend({
   // Use this to remove Views, internally uses `getViews` so you can pass the
   // same argument here as you would to that method.
   removeView: function(fn) {
+    var views;
+
     // Allow an optional selector or function to find the right model and
     // remove nested Views based off the results of the selector or filter.
-    return this.getViews(fn).each(function(nestedView) {
+    views = this.getViews(fn).each(function(nestedView) {
       nestedView.remove();
     });
+
+    // call value incase the chain is evaluated lazily to ensure the views get
+    // removed
+    views.value();
```

Comment:
so glad @jdalton chimed in to point out the obvious. this line makes `removeView` work in both underscore and lodash again